components/root/root.jsx

```diff
@@ -96,6 +95,11 @@ const LoggedInRoute = ({component: Component, ...rest}) => (
     />
 );
 
+const desktopMediaQuery = window.matchMedia(`(min-width: ${Constants.DESKTOP_SCREEN_WIDTH + 1}px)`);
```

Comment:
Just double checking that `window.matchMedia` is supported on all the browsers (+electon) we support? I only ask because `window.matchMedia` is unfamiliar to me and I know we've had issues with browser API support before.

Reply:
Good question! I did a [cursory check](https://caniuse.com/?search=MediaQueryList) before using and support is surprisingly good! A more detailed look shows that a more recent version of Safari (less than v14) needed to use `addListener` instead of `addEventListener`. Updated to support fallbacks including the throttled window.resize event.